src/diff/props.js

```diff
@@ -57,39 +56,34 @@ function setStyle(style, key, value) {
  * @param {boolean} isSvg Whether or not this DOM node is an SVG node or not
  */
 export function setProperty(dom, name, value, oldValue, isSvg) {
-	let s, useCapture, nameLower;
+	let useCapture, nameLower;
 
-	if (isSvg) {
-		if (name === 'className') {
-			name = 'class';
-		}
-	} else if (name === 'class') {
-		name = 'className';
-	}
+	if (isSvg && name == 'className') name = 'class';
```

Comment:
I'm curious if anyone is opposed to this change. It means we would be using `setAttribute("class")` for a `class` prop and `.className=` for a `className` prop. They're the same thing, but it's possible there's a performance difference.